Lib/_pyio.py

```diff
@@ -766,7 +766,7 @@ def truncate(self, pos=None):
 
     def flush(self):
         if self.closed:
-            raise ValueError("flush of closed file")
+            raise ValueError("flush on closed file")
```

Comment:
I considered this a typo mainly due to the wording used on other methods such as `tell`, `truncate`, `seek` and others. I might not see it but, is there anything special about `flush` (that doesn't apply to the other methods) that warrants the use of `of` instead of `on`?